satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -2721,7 +2988,8 @@ datasets:
     resolution: 32000
     file_type: nc_fci_asr
     file_key: pixel_percentage
-    long_name: pixel_percentage_cloudy
+    long_name: Percentage of FoR pixels used (cloudy pixels)
+    standard_name: pixels_used_fraction
```

Comment:
also add `units: %`, set to none in netcdf file.

Reply:
done